live-build/misc/ansible-roles/appliance-build.zfsonlinux-development/tasks/main.yml

```diff
@@ -65,6 +65,39 @@
     accept_hostkey: yes
     update: no
 
+- shell: ls /lib/modules
+  register: kernel_versions
+
+- shell: |
+    CONFIG_OPTS=" \
+    --build=x86_64-linux-gnu \
+    --prefix=/usr \
+    --includedir=/usr/include \
+    --mandir=/usr/share/man \
+    --infodir=/usr/share/info \
+    --sysconfdir=/etc \
+    --localstatedir=/var \
+    --libdir=/usr/lib/x86_64-linux-gnu \
+    --libexecdir=/usr/lib/x86_64-linux-gnu \
+    --disable-maintainer-mode \
+    --bindir=/usr/bin \
+    --sbindir=/sbin \
+    --libdir=/lib \
+    --with-python=3 \
+    --with-udevdir=/lib/udev \
+    --with-systemdunitdir=/lib/systemd/system \
+    --with-systemdpresetdir=/lib/systemd/system-preset \
+    --with-linux=/lib/modules/{{ kernel_versions.stdout_lines[0] }}/build \
+    --with-linux-obj=/lib/modules/{{ kernel_versions.stdout_lines[0] }}/build"
+    cd /export/home/delphix/zfs
```

Comment:
I think you can add:
```
args:
  chdir: /export/home/delphix/zfs
```
and remove this command.. not that it really matters either way, though.